common/server_protocolhandler.cpp

```diff
@@ -821,7 +821,7 @@ Server_ProtocolHandler::cmdCreateGame(const Command_CreateGame &cmd, Server_Room
     Server_Game *game = new Server_Game(
         copyUserInfo(false), gameId, description, QString::fromStdString(cmd.password()), cmd.max_players(), gameTypes,
         cmd.only_buddies(), onlyRegisteredUsers, cmd.spectators_allowed(), cmd.spectators_need_password(),
-        cmd.spectators_can_talk(), cmd.spectators_see_everything(), room);
+        cmd.spectators_can_talk(), cmd.spectators_see_everything(), cmd.starting_life_total(), room);
```

Comment:
I don't think this is backwards compatible this way, it needs a default value in case the cmd does not have starting life total set like an old client would, by the use of cmd.has_starting_life_total

Reply:
Adressed in [e068302](https://github.com/Cockatrice/Cockatrice/pull/5174/commits/e06830209c487f2e38130640ea82880068646069)